.github/workflows/run_script_minkabu.yml

```
@@ -12,17 +12,22 @@ permissions:
 jobs:
   run_script:
     runs-on: ubuntu-latest
+    env:
+      ARTIFACT_DIR: ${{ runner.temp }}/artifacts/minkabu
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)